index.js

```diff
@@ -17,11 +21,19 @@ module.exports = function (source) {
 		return;
 	}
 
-	var compiler = new KaitaiStructCompiler();
-	var kaitaiImport = 'var KaitaiStream = require("kaitai-struct/KaitaiStream")\n';
+	const context = this.context;
```

Comment:
I would also rename this to something more explanatory than the broad term `context` - maybe `moduleDir`?